# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/painting/crashtests/1819957-1.html
# Commit: 5ded58486aba
# Full Hash: 5ded58486abab5712f305a88bbdd746ae74dcdef
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-03-06 09:45:20
# Description:
#   Bug 1819957 - Split out nsTextPaintStyle into its own source file. r=emilio
#   
#   This lets us instantiate it on the stack in nsDisplayText::Paint.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171635
# ==============================================================================

diff -r 58235f400c11 -r 5ded58486aba layout/painting/crashtests/1819957-1.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/painting/crashtests/1819957-1.html	Sun Mar 05 22:24:10 2023 +0000
@@ -0,0 +1,14 @@
+<style>
+::selection {
+  text-shadow: 0 1px 0px -moz-mac-menushadow;
+}
+</style>
+<script>
+document.addEventListener('DOMContentLoaded', () => {
+  document.execCommand("selectAll", false)
+})
+</script>
+<dir>
+<menu id="foo"></menu>
+<menu>
+<link>a</link>